### PR TITLE
Minor changes to Questions wording

### PR DESCRIPTION
**Pull request by sebastianbarry**
demo-survey-v1.xml
- Added an "Other" option to "Which_best_describes_your_prim"
- Added a "neither" option to "Do_you_own_or_rent_your_home"
- Changed the wording on "Do_you_work_for_either_pay_or_" to "Are you a paid worker?" to be more clear

### CODE REVIEW

File: survey-resources/data-xml/demo-survey-v1.xml [hunk 1]
```diff
@@ -716,7 +722,7 @@
             <value>Prefer not to say</value>
           </text>
           <text id="/arcEm5iPB4F9CQZR258k4r/group_hg4zz25/Do_you_work_for_either_pay_or_:label">
-            <value>Do you work for either pay or profit?</value>
+            <value>Are you a paid worker?</value>
```
Comment (shankari) [inline survey-resources/data-xml/demo-survey-v1.xml:725]: Need to change this in Spanish as well

**Review by shankari — CHANGES_REQUESTED** [review]
You need to also add the new options to the select1 entries at the end of the survey.
See https://github.com/e-mission/e-mission-phone/commit/70c2d2986b67844f4c171c5b8d12638f75429e9a#diff-25927b5a1c09baa620ed049840064c39222b70c2296afddd7c6ef6965e899184R1569

You have only added new text labels at this point.

File: survey-resources/data-xml/demo-survey-v1.xml [hunk 2]
```diff
@@ -161,6 +161,9 @@
           <text id="/arcEm5iPB4F9CQZR258k4r/group_pa5ah98/Do_you_own_or_rent_your_home/rent:label">
             <value>Alquilado</value>
           </text>
+          <text id="/arcEm5iPB4F9CQZR258k4r/group_pa5ah98/Do_you_own_or_rent_your_home/rent:label">
+            <value>Ninguno</value>
+          </text>
```
Comment (shankari) [inline survey-resources/data-xml/demo-survey-v1.xml:166]: also you need to change the text for the label, so this needs to be `....home/neither:label`
and then you need to add a new select option for `neither`

**Review by shankari — CHANGES_REQUESTED** [review]
see my code around adding a new `prefer_not_to_say` option for `How_many_days_do_you_usually_w_001` in

https://github.com/e-mission/e-mission-phone/commit/b973ad78e1832424041a4cdc53540b1b156538d7

for an example

**Comment by sebastianbarry** [comment]
Thanks for the changes! I was able to get it working, see below
Screen Shot 2022-07-11 at 3 06 42 PM
Screen Shot 2022-07-11 at 3 06 31 PM
Screen Shot 2022-07-11 at 3 06 21 PM

**Review by shankari — APPROVED** [review]

**Pull request merged**